concent_api/core/views.py

```diff
@@ -17,6 +21,8 @@
 @api_view
 @require_POST
 def send(_request, client_message):
+    if client_message:
```

Comment:
Also, `client_message` should never be `None` here (please add an `assert` that checks that). If `api_view` gets `None` after decoding a message, it should return an error. But this is not related to this branch. Please report and fix it separately.

Reply:
Yes, lets make a separate issue out of it. Generally I think it should never hit the view and fail directly in api_views.py. I will work on that later.